draft-ietf-quic-transport.md

```diff
@@ -1069,6 +1069,17 @@ to cease using the connection IDs when requested can result in connection
 failures, as the issuing endpoint might be unable to continue using the
 connection IDs with the active connection.
 
+An endpoint MAY elect to limit the number of outstanding RETIRE_CONNECTION_ID
```

Comment:
I think this should be a SHOULD.

In case of a client-driven attack using path probes, an innocent server without this type of bound would end up holding infinite amount of state.

Reply:
SHOULD is fine with me.